docs/ref/validators.txt

```diff
@@ -159,6 +159,35 @@ to, or in lieu of custom ``field.clean()`` methods.
         validation, so you'd need to add them to the ``allowlist`` as
         necessary.
 
+``DomainNameValidator``
+-----------------------
+
+.. versionadded:: 5.1
+
+.. class:: DomainNameValidator(message=None, code=None, inverse_match=None, flags=0, accept_idna=True)
```

Comment:
```suggestion
.. class:: DomainNameValidator(accept_idna=True, message=None, code=None)
```
I think we wouldn't expect the user to set `inverse_match` and `flags`. This is then similar to the URLValidator definition
```
.. class:: URLValidator(schemes=None, regex=None, message=None, code=None)
```